### DISCUSSION TITLE
想问一下，这个开源项目最后是如何把RSS的JS对象转换成XML文本的？

### DISCUSSION THREAD

**Discussion by good1uck** [DIYgod/RSSHub#16187] [Q&A] [answered]
### 这是一个什么样的功能？

想问一下，这个开源项目最后是如何把RSS的JS对象转换成XML文本的？
从公开的制作路由教程信息中可以知道，最后会return一个包含了RSS信息的对象，
但是阅读完相关的源码后依然没有发现这个JS对象是如何被进一步解析成XML的。望告知

### 这个功能可以解决什么问题？

想问一下，这个开源项目最后是如何把RSS的JS对象转换成XML文本的？

### 这不是重复的功能请求和 RSS 提案

- [X] 我已经搜索了 [现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保这项功能尚未被请求及不是 [新的 RSS 提案](https://github.com/DIYgod/RSSHub/issues/new?assignees=&labels=RSS+proposal&template=rss_request_zh.yml)。

**Comment by TonyRL** [comment, marked as answer]
https://github.com/DIYgod/RSSHub/blob/42a10b9198d3e3c28847f034c0fe0df4f9a4e129/lib/middleware/template.tsx#L101-L112

**Reply by good1uck** [reply]
多谢
